### DISCUSSION TITLE
Search order in topics sorted by "best match" and "most stars" are the same

### DISCUSSION THREAD

**Discussion by eluc1dat0r** [community/community#36635] [Other Feature Feedback, Questions, & Ideas]
As described in the title. For example the topic ecommerce (link below), it doesn't matter if I use best match or most stars, I get the same order as a result. But for the search term ecommerce the best match filter is working.
Is this intended?

https://github.com/topics/ecommerce
https://github.com/search?q=ecommerce

**Comment by ossanna16** [comment]
Hey there @eluc1dat0r and thank you for asking a good question! I'm glad you found our community 🙂
